v2/brigadier/src/jobs.ts

```diff
@@ -50,3 +110,47 @@ export class JobHost {
   public os?: string
   public nodeSelector: { [key: string]: string } = {}
 }
+
+export class ParallelJob extends Job {
+  constructor(private readonly jobs: Job[]) {
+    super();
+  }
+
+  public async run(): Promise<void> {
+    const promises = this.jobs.map((job) => job.run());
+    await Promise.all(promises);
+  }
+}
+
+export class SequentialJob extends Job {
+  constructor(private readonly jobs: Job[]) {
+    super();
+  }
+
+  public async run(): Promise<void> {
+    for (const job of this.jobs) {
+      await job.run();
+    }
+  }
+}
+
+export class RetryableJob extends Job {
```

Comment:
`RetryableJob` does indeed make sense to me as a name for this thing, given the above ETL-isms and stuff. So I very much like both this name and this addition to the API.